docs/setup/storage/blob.md

```diff
@@ -10,23 +10,23 @@ has_children: false
 
 # Prepare Your Blob Storage Container
 
-[Create a container in Azure portal](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal#create-a-container):
+[Create a container in the Azure portal](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal#create-a-container):
 
-1. From the Azure portal, Storage Accounts, choose your account, then in the container tab click `+ Container`.
+1. From the Azure portal, go to Storage Accounts and choose your account. Next, click `+ Container` in the container tab.
 1. Make sure you block public access
 
-## Authenticate with Secret Key
+## Authenticate with the Secret Key
 {: .no_toc }
 
 If you want lakeFS to authenticate with your storage using the storage account key, go to the `Access Keys` tab and click `Show Keys`. Use the values under `Storage account name` and `Key` in the [lakeFS configuration](../../deploy/azure.html#on-azure-vm).
 
 ## Authenticate with Active Directory
 {: .no_toc }
 
-In case you want your lakeFS Installation (we will install in the next step) to access this Container using Active Directory authentication,
+In case you want your lakeFS Installation to access this Container using Active Directory authentication (we will install in the next step),
 First go to the container you created in step 1.
```

Comment:
```suggestion
In case you want your lakeFS Installation to access this Container using Active Directory authentication,
first go to the container you created in step 1.
```